clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -5746,11 +5747,11 @@ void Clang::ConstructJob(Compilation &C, const JobAction &JA,
         CmdArgs.push_back("-fsycl-enable-int-header-diags");
       }
 
-      // Add the -include-footer option to add the integration footer
+      // Add the -include-internal-footer option to add the integration footer
```

Comment:
Nit: Add the comment inside the 'if' construct so that it reads similar to the 
'header' option?